topics/introduction/metadata.yaml

```diff
@@ -302,6 +302,23 @@ material:
     contributors:
       - jmchilton
       - erasche
+  -
+    title: "Jupyter Interactive environment"
+    type: "tutorial"
+    name: "galaxy-intro-jupyter"
+    zenodo_link: ""
+    workflows: no
+    galaxy_tour: no
+    hands_on: yes
+    slides: no
+    questions:
+      - "How to use a Jupyter Notebook in Galaxy"
+    objectives:
+      - "Learn about the Jupyter Interactive Environment"
+    time_estimation: "30 min"
+    key_points:
```

Comment:
could you add some keypoints here? (just quick list of things learnt here, for example that you can install libraries in jupyter, load/save datasets from history with special commands, etc)